### DISCUSSION TITLE
Forward/back navigation

### DISCUSSION THREAD

**Discussion by ThatAdamGuy** [athensresearch/athens#1260] [Feature Requests]
I'd like to be able to quickly jump back or forward a page (just like in a web browser)...
- using keyboard shortcuts
- and ideally using my mouse forward/back buttons, too!

Thanks for considering this :)

**Comment by shanberg** [comment]
@ThatAdamGuy Could you let us know what operating system you're using? Thanks!

**Comment by ThatAdamGuy** [comment]
Hi Stuart,

I'm on Mac Big Sur v11.4

**Comment by julionav** [comment]
Interesting. This works for me on Ubuntu using the back and forward mouse buttons.

Does `option+left ` (nav back) and `option+right` (nav forward) work for you?

**Comment by ThatAdamGuy** [comment]
Weird!
My mouse buttons work in most other apps but not Athens.
However, option+left and option+right work for me in Athens.
